.github/workflows/ci-windows-artifacts.yml

```diff
@@ -46,7 +43,18 @@ jobs:
     - name: Build libyuv
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
-      run: ./libyuv.cmd
+      # Use clang-cl to build libyuv. The assembly code in libyuv is written in the
+      # GCC inline assembly format. Visual C++ doesn't support GCC inline assembly.
+      run: >
+        set "CC=clang-cl"
```

Comment:
Nit: The quotes can be omitted. I remember the quotes are used in appveyor.yml because we connect two `set` commands with `&&`:

```
set "CC=clang-cl" && set "CXX=clang-cl"
```

Reply:
OK, in PowerShell these two lines should read:

```
$Env:CC = 'clang-cl'

$Env:CXX = 'clang-cl'
```